packages/app/components/card/rows/elements/ownership.tsx

```diff
@@ -36,30 +34,29 @@ function OwnershipContainer({
 }
 
 export function Ownership({ nft }: Props) {
-  const isDarkMode = useIsDarkMode();
-  const { data, loading } = useNFTDetails(nft?.nft_id);
+  const { data } = useNFTDetails(nft?.nft_id);
 
   if (!nft) return null;
 
-  if (loading) {
-    return (
-      <OwnershipContainer count={nft.owner_count}>
-        {Array(nft.owner_count)
-          .fill(0)
-          .slice(0, 4)
-          .map((_, index) => (
-            <Skeleton
-              key={`nft-${nft.nft_id}-owner-${index}-skeleton`}
-              width={14}
-              height={14}
-              colorMode={isDarkMode ? "dark" : "light"}
-              radius="round"
-              show={true}
-            />
-          ))}
-      </OwnershipContainer>
-    );
-  }
+  // if (loading) {
```

Comment:
drop card no longer has this data, if added this will every time adjust the card height.
so if has `Owner` we should show it directly.